recipes/scijava-jupyter-kernel/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -euo
+
+# Install Beakerx...
+# ...waiting for a Maven artifact.
+cd $RECIPE_DIR/
+
+git clone --depth 1 https://github.com/twosigma/beakerx.git
```

Comment:
Does it make sense to pin the git checkout to a particular release of beaker, or do you always need the latest?